ascii.libsonnet

```
@@ -26,4 +26,10 @@ local d = import 'github.com/jsonnet-libs/docsonnet/doc-util/main.libsonnet';
     [d.arg('c', d.T.string)]
   ),
   isNumber(c): std.isNumber(c) || (cp(c) >= 48 && cp(c) < 58),
+
+  '#isStringNumeric':: d.fn(
+    '`isStringNumeric` reports whether string `s` consists only of numeric characters.',
+    [d.arg('str', d.T.string)]
+  ),
+  isStringNumeric(str): std.all(std.map(self.isNumber, std.stringChars(str)),
```

```suggestion
  isStringNumeric(str): std.all(std.map(self.isNumber, std.stringChars(str))),
```
I think one parentheses is missing